recipes/django-rosetta/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "django-rosetta" %}
+{% set version = "0.9.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0315a6406818c51f0f0fe04aa30328e802c8befcab1d3a34a242961284cb3387
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - rosetta
+
+about:
+  home: https://pypi.org/project/django-rosetta
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A Django application that eases the translation of Django projects'
+
+  description: |
+    Rosetta is a Django application that eases the translation process of your Django projects.
+    Because it doesn't export any models, Rosetta doesn't create any tables in your project's database. 
+    Rosetta can be installed and uninstalled by simply adding and removing a single entry in your project's INSTALLED_APPS and a single line in your main urls.py file
+    Note: As of version 0.7.13 django-rosetta requires Django 1.8 or later. As of version 0.9.0, django-rosetta requires Django 1.11 or later.
+  
+  doc_url: http://django-rosetta.readthedocs.org/
+  dev_url: https://github.com/mbi/django-rosetta/
+
+extra:
+  recipe-maintainers:
+    - mbi
```

Comment:
CC @mbi - are OK with being a maintainer?

Reply:
I don't own a Windows machine, nor have ever used Conda-Forge, so I can't say I'm ready to take on this commitment  :man_shrugging:

Reply:
OK! @kcemenike - please remove @mbi from this list

Reply:
done, thanks